client/index.js

```diff
@@ -159,7 +159,7 @@ async function doRender ({ Component, props, hash, err, emitter: emitterProp = e
 
 let isInitialRender = true
 function renderReactElement (reactEl, domEl) {
-  if (isInitialRender) {
+  if (isInitialRender && typeof ReactDOM.hydrate === 'function') {
```

Comment:
Why do we need to check this? This came with `16.0.0` right?

Reply:
@arunoda to support `preact` and the likes that haven't added `hydrate` yet

Reply:
Ah ha. Shall we add a comment here about this?

Reply:
Yeah, good idea 👍